vendor/modules.txt

```diff
@@ -71,6 +71,15 @@ github.com/go-openapi/jsonpointer
 github.com/go-openapi/jsonreference
 # github.com/go-openapi/swag v0.19.14
 github.com/go-openapi/swag
+# github.com/gobwas/glob v0.2.3
+github.com/gobwas/glob
```

Comment:
the last commit to this library was >3 years ago. Are you confident it is well-maintained?